.gitlab/ci.yml

```diff
@@ -95,6 +96,8 @@ tarball suse-42.1:
         # build tools for CodeCompass
         - zypper install -y -t pattern devel_basis
         - zypper install -y binutils gcc-c++ gmp-devel
+        # build tools for OpenLDAP
+        - zypper install -y groff groff-full
```

Comment:
Is `groff-base` not sufficient, like for Ubuntu?

Reply:
There is no supported groff-base package for SUSE 42 and 15 but groff is enough so I removed groff-full.